Logfile initialized
[INFO] logfile = D:\Workspaces\Titanium\txde\build.log
[DEBUG] D:\Program File\Titanium\mobilesdk\win32\3.1.2.GA\android\builder.py install TXDE D:\Program File\Android-SDK\sdk D:\Workspaces\Titanium\txde com.netsky.txde 5
[INFO] Titanium SDK version: 3.1.2 (08/14/13 12:45 5ceaff8)
[DEBUG] Waiting for device to be ready ...
[ERROR] Exception occured while building Android project:
[ERROR] Traceback (most recent call last):
[ERROR]   File "D:\Program File\Titanium\mobilesdk\win32\3.1.2.GA\android\builder.py", line 2621, in <module>
[ERROR]     builder.build_and_run(True, avd_id, device_args=device_args, debugger_host=debugger_host, profiler_host=profiler_host)
[ERROR]   File "D:\Program File\Titanium\mobilesdk\win32\3.1.2.GA\android\builder.py", line 2155, in build_and_run
[ERROR]     self.wait_for_device('d')
[ERROR]   File "D:\Program File\Titanium\mobilesdk\win32\3.1.2.GA\android\builder.py", line 526, in wait_for_device
[ERROR]     devices = self.sdk.list_devices()
[ERROR]   File "D:\Program File\Titanium\mobilesdk\win32\3.1.2.GA\android\androidsdk.py", line 300, in list_devices
[ERROR]     raise Exception(err)
[ERROR] Exception: error: 
[ERROR] 
